Restructure

kobejean opened the pull request:
@Jacoab I finally got the project restructured to work as a package.
ttrends is the name of the package and the tests and scripts are directories that contains code that use the package. If you are happy with this structure let’s merge it with the master branch.

kobejean committed:
Added color to output text

kobejean committed:
Removed annoying print message

kobejean committed:
A mess I haven't fixed

Trying to turn the project into a package but I’m making a mess

kobejean committed:
Restructured as a package called "trends"

kobejean committed:
Fixed Travis CI Problem

kobejean committed:
Added data directory in scripts directory

kobejean committed:
FUCTStreamTransformer added condition of at least one url to priority

kobejean committed:
Improved FUCTStreamTransformer priority function

now uses a tuple as priority output(at_least_1_url, followers_count)

kobejean committed:
Asks if you want to see the output data when stream ends

kobejean committed:
Changed collect_count to sample_size

kobejean committed:
Update .gitignore

kobejean committed:
Documentation and readability

kobejean committed:
Update documentation

Jacoab commented:
This all looks good, I like the ttrends package import.  The only thing i'd prefer is if we call the scripts directory src.  Its a bit nit picky but that's usually the convention for storing your source code.

Jacoab reviewed (changes requested):
Everything looks good, the only thing i think we should change is the scripts directory to src.  src is the general unix naming convention.

kobejean committed:
Changes scripts to src

kobejean committed:
Commented out unused streamListener methods

I’m under the suspicion that I am causing a few errors for not
implementing them properly.

kobejean commented:
Alright I've changed it to src
